packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -322,6 +322,26 @@ export class TimeSpan {
     return hhmmss;
   }
 
+  getYoutubeFormatTimestamp() {
+    const hours = Math.floor(this.getHours());
+    const minutes = Math.floor(this.getMinutes()) % 60;
+    const seconds = Math.floor(this.getSeconds()) % 60;
+    let timestamp = '';
+
+    const formattedSeconds = seconds < 10 ? '0' + seconds : seconds;
+
+    if (hours > 0) {
+      const formattedMinutes = minutes < 10 ? '0' + minutes : minutes;
+      timestamp += hours + ':' + formattedMinutes + ':' + formattedSeconds;
+    } else {
+      const formattedMinutes =
+        minutes === 0 ? '0' : minutes < 10 ? '0' + minutes : minutes;
+      timestamp += formattedMinutes + ':' + formattedSeconds;
+    }
+
+    return timestamp;
+  }
+
```

Comment:
Here you could probably use string literals and a helper function to simplify this logic and avoid having to declare `formattedMinutes` twice. What do you think?

```suggestion
  getYoutubeFormatTimestamp() {
  const format = (num) => (num < 10 ? '0' + num : num);

  const hours = Math.floor(this.getHours());
  const minutes = Math.floor(this.getMinutes()) % 60;
  const seconds = Math.floor(this.getSeconds()) % 60;

  const formattedSeconds = format(seconds);
  const formattedMinutes = format(minutes);

  if (hours > 0) {
    return `${hours}:${formattedMinutes}:${formattedSeconds}`;
  } else {
    return `${minutes === 0 ? '0' : formattedMinutes}:${formattedSeconds}`;
  }

```

Reply:
Could be yeah, I don't see any impact in going one way or another,
personally, not a big fan of declaring  a function inside a method tho.

Reply:
then you could declare it outside X) I just think its a bit more readable IMO

Reply:
That's a class we are dealing with, declaring it outside as a method? no we don't wont expose that in this class.
Declaring it as an orphan function in the file? could be but what does it add?
Anyway we are having a discussion about an extremely  minor thing, that mainly boils down to preference but does not have a real impact.

Reply:
Also as a general rule, I don't think minor things like this could be stoppers to not approve PRs, 
mainly to avoid doing a lot of back and forth on PRs before getting them merged.

If there are things impactful that can be improved yes of course they are the most welcome,
For very minor things where doing something can be done using method A or method B but without a real impact if any, yes they also are the most welcome to suggest, but should not be stoppers from approving PRs.
That's my opinion and that's the logic I follow at least to approve PRs or not,

Reply:
I was mostly kidding for the function, but I agree with you that functions inside of class methods are usually not the best practice. In this case however, it was such a small and simple function that I thought it was making the method more readable than not. 

Anyways, as you said, this is definitely not a blocker for me, but I do think readability to be impactful.